src/Components/ModalSpinner.tsx

```diff
@@ -1,22 +1,14 @@
 import { useEffect, useRef } from 'react'
-import { Modal } from 'react-bootstrap'
 import { connect } from 'scrivito'
 
 export const ModalSpinner = connect(function ModalSpinner() {
   const dialogRef = useRef<HTMLDialogElement>(null)
   useEffect(() => dialogRef.current?.showModal(), [])
 
   return (
-    <>
-      <dialog className="d-none" ref={dialogRef} />
-      <Modal
-        backdrop="static"
-        centered
-        dialogAs={() => <div className="loader" />}
-        fullscreen
-        keyboard={false}
-        show
-      />
-    </>
+    <dialog className="loader-dailog" ref={dialogRef}>
```

Comment:
* [ ] Now that `Modal` no longer handles the keyboard input, we must prevent closing the dialog with Escape.